satellite/metainfo/config.go

```diff
@@ -57,6 +57,10 @@ func NewStore(logger *zap.Logger, dbURLString string) (db storage.KeyValueStore,
 		err = Error.New("unsupported db scheme: %s", driver)
 	}
 
+	if err != nil {
```

Comment:
Again, this check is probably just extra code / against Go style guidelines.

Reply:
This one it's needed as the `log.Debug` should only happen if there is no error.